whi_motion_teleop:
  command_frequency: 20 # Hz
  motion_state_topic: NaviBOT/motion_state
  sw_estop_topic: estop
  rc_state_topic: rc_state
  linear: # m/s
    min: 0.08
    max: 0.3
    step: 0.01 
  angular: # rad/s
    min: 0.1
    max: 1.0
    step: 0.05
  